packages/nimble-components/src/table-column/base/tests/table-column.stories.ts

```diff
@@ -456,3 +456,142 @@ export const sorting: StoryObj<SortingTableArgs> = {
         }
     }
 };
+
+interface ColumnWidthTableArgs extends SharedTableArgs {
+    fractionalWidth: ExampleColumnFractionalWidthType;
+    minPixelWidth: number;
+    getColumnWidthData: (
+        columnId: string,
+        args: ColumnWidthTableArgs
+    ) => number | undefined;
+}
+
+const fractionalWidthOptions = {
+    [ExampleColumnFractionalWidthType.default]: [],
+    [ExampleColumnFractionalWidthType.firstColumnHalfSize]: [
+        {
+            columnId: 'first-name-column',
+            width: 0.5
+        }
+    ],
+    [ExampleColumnFractionalWidthType.firstColumTwiceSize]: [
+        {
+            columnId: 'first-name-column',
+            width: 2
+        }
+    ],
+    [ExampleColumnFractionalWidthType.thirdColumnHalfFourthColumnTwice]: [
+        {
+            columnId: 'favorite-color-column',
+            width: 0.5
+        },
+        {
+            columnId: 'quote-column',
+            width: 2
+        }
+    ]
+} as const;
+
+const fractionalWidthDescription = `Some table columns (like \`nimble-table-column-text\`) can be configured to have a fractional width such that its width
+                                    will be proportional to other columns also configured with a fractional width (which is the default). So,
```

Comment:
> (which is the default)

This isn't necessarily true from a column API perspective, right? A given column will decide what is the default and whether it even supports both fractional & fixed sizing, right?

Reply:
Yeah, I can see the confusion. I think what I'm trying to convey here, is that even columns that don't expose the `fractional-width` API to control their fractional-width, will still be sized in response to a table size (or by another column changing its `fractional-width`) in a way that mirrors it having a `fractional-width` of 1.

Reply:
Updated comment.